.doc_gen/metadata/support_metadata.yaml

```diff
@@ -182,6 +182,15 @@ support_Scenario_GetStartedSupportCases:
     - Get and display a list of resolved cases for the current day.
   category: Scenarios
   languages:
+    Java:
```

Comment:
You'll want to add the sections for the Hello Support and each of the single action examples also.